src/main/java/org/alliancegenome/curation_api/interfaces/base/BaseUpsertControllerInterface.java

```diff
@@ -15,5 +16,5 @@ public interface BaseUpsertControllerInterface<E extends AuditedObject, T extend
 	@POST
 	@Path("/upsert")
 	@JsonView(View.FieldsOnly.class)
-	E upsert(T dto) throws ObjectUpdateException;
+	E upsert(T dto) throws ValidationException, KnownIssueValidationException;
```

Comment:
So this should be able to just throw ValidationException?

Reply:
yep, missed that one - will update